src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -119,7 +119,29 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
   --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID } from "react-native-appwrite";
 
+  const client = new Client()
+      .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+      .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+      '[BUCKET_ID]',
+      ID.unique(),
+      { name: "file.jpeg", type: "image/jpeg", uri: "file:///data/user/0/host.exp.exponent/cache/ExperienceData/text.jpeg", size: 183799}, // FILE
+      ["read(\"any\")"]
```

Comment:
Let's use the [Permission helper](https://github.com/appwrite/sdk-for-react-native/blob/cc2ccdc99dd170eb393d706665f983d416295e77/src/permission.ts#L3) for this